docs/TipsAndTricks.md

```diff
@@ -0,0 +1,24 @@
+# Tips and Tricks
+
+**How to get an error in your console too:**
+
+The `Redbox` errors are great to clearly see rendering issues, and avoiding an uncaught error from breaking your app.  But there are some advantages to a thrown error in the console too, like filename resolution via sourcemaps, and click-to-open.  To get the best of both worlds, modify your app entry point as follows:
+
+```jsx
+import Redbox from 'redbox-react';
+
+const consoleErrorReporter = ({error}) => {
+  console.error(error);
+  return <Redbox error={error} />;
+};
+
+consoleErrorReporter.propTypes = {
+  error: React.PropTypes.error
+};
+
+render((
+  <AppContainer errorReporter={consoleErrorReporter}>
+    <AppRoot />
+  </AppContainer>
+), document.getElementById('react-root'));
+```
```

Comment:
Hi,

thanks for the `consoleErrorReporter` tip!

I tried to use it as described above and have a few questions/remarks:

- I had to add `redbox-react` to my `package.json`. If that is desired, I think it would make sense to add it to the docu
- `React.PropTypes.error` raised an error: `prop type 'error' is invalid`. I changed it to the proptype definition from Redbox (`React.PropTypes.instanceOf(Error).isRequired`) then it works. Not sure if this is a problem with my React version?
- Might not be directly related but maybe someone knows: What is the best setting of `devtool` in my `webpack.config.js` file? I tried several ones but the stacktraces dumped both by `Redbox` and from `console.error` doesn't point to my source files (both link to my `bundle.js` that has been generated by webpack, like this: `http://localhost:3000/dist/bundle.js:36376:12`). When running my app without hotloader (just removing the `AppContainer` wrapper, no changes to `.babelrc` or `webpack.config.js`) all errors dumped on Chrome's devtool console are correctly linked to my source files (something like `webpack:///./client/src/components/editor/ContactEditor.js`) 

My setup: RHL 3.0.0beta, React 15, npm 2.14.12, babel 6 (with presets es2015, stage-0, react and plugin react-hot-loader/babel), webpack-dev-server 1.14, Chrome 50.0, MacOS

Again: thanks for your efforts!

Nils